demisto_sdk/commands/common/errors.py

```diff
@@ -1135,7 +1135,6 @@ def release_notes_not_finished():
     def release_notes_file_empty():
         return "Your release notes file is empty, please complete it\nHaving empty release notes " \
                "looks bad in the product UI.\nIf the change you made was minor, please use " \
-               "\"Maintenance and stability enhancements.\" for general changes, or use " \
```

Comment:
Instead of "If the change you made was minor" in the line above this, let's write:
```
Make sure the release notes explicitly describe what changes were made, even if they are minor.
```